### PR TITLE
fix: specify ASM bucket (ASM plugin)

### PR DESCRIPTION
**Pull request by 0marperez**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt [hunk 1]
```diff
@@ -14,7 +14,7 @@ import org.gradle.kotlin.dsl.withType
 
 private const val TASK_GROUP = "Verification"
 internal const val OUTPUT_PATH = "reports/metrics/"
-internal const val S3_ARTIFACT_SIZE_METRICS_BUCKET = "" // TODO: FILL IN BUCKET
+internal const val S3_ARTIFACT_SIZE_METRICS_BUCKET = "artifact-size-metrics-aws-sdk-kotlin"
```
Comment (lauzadis) [inline build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt:17]: question/naming: will this bucket will also contain the metrics for smithy-kotlin (and possibly aws-crt-kotlin)? Maybe the name can be made more generic to reflect that

**Review by lauzadis — APPROVED** [review]

File: build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt (hunk 1 above)
Comment (lauzadis) [reply build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt:17]: Do we need to specify the team name if it's always going to be in our team account? Why not just `artifact-size-metrics`?

Comment (0marperez) [reply build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt]: I thought the bucket name would've been taken already. But this works better. Good call :+1:

File: build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt [hunk 2]
```diff
@@ -14,7 +14,7 @@ import org.gradle.kotlin.dsl.withType
 
 private const val TASK_GROUP = "Verification"
 internal const val OUTPUT_PATH = "reports/metrics/"
-internal const val S3_ARTIFACT_SIZE_METRICS_BUCKET = "" // TODO: FILL IN BUCKET
+internal const val S3_ARTIFACT_SIZE_METRICS_BUCKET = "artifact-size-metrics"
```
Comment (ianbotsf) [inline build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/ArtifactSizeMetricsPlugin.kt:17]: **Nit**: Let's leave a short comment indicating which account owns this bucket

**Review by ianbotsf — APPROVED** [review]

**Review by ianbotsf — APPROVED** [review]

**Pull request merged**